cirq/protocols/approximate_equality_protocol.py

```diff
@@ -94,6 +95,12 @@ def approx_eq(val: Any, other: Any, *, atol: Union[int, float] = 1e-8) -> bool:
     if isinstance(val, str):
         return val == other
 
+    if isinstance(val, sympy.Basic) or isinstance(other, sympy.Basic):
+        delta = sympy.Abs(other - val).simplify()
+        if not delta.is_number:
+            return False
```

Comment:
We should throw an error, rather than False - they might be equal but we don't know!

Reply:
Done